tests/helpers.py

```diff
@@ -863,3 +863,31 @@ def action_open(self, namespace, repository):
         with Replace('subprocess.Popen', self.Popen):
             self.service.open(user=namespace, repo=repository)
 
+    def action_review(self, namespace, repository, branch):
```

Comment:
I am not sure about this. Shall we keep it in `helpers` or it is better to move it to the gerrit test itself? Or may we try to use `action_request_create` instead?